Gemfile

```
@@ -1,6 +1,7 @@
 source "https://rubygems.org"
 
 gem "github-pages", group: :jekyll_plugins
+gem "webrick" if Gem.win_platform?
```

webrick also needs to be added here for other OSes (confirmed on my Mac + macOS Big Sur 11.5.2)

```suggestion
gem "webrick"
```